doc/source/datainputs.rst

```diff
@@ -16,28 +16,128 @@ To simplify the examples below, we will assume in all cases that the user wishes
+Other attributes, such as particle orientations, are not included automatically in the system representation and must be loaded as separate NumPy arrays.
 
 GSD Trajectories
-================
+----------------
 
-Using the GSD Python API, GSD files can be very easily integrated with **freud** as shown in :ref:`gettingstarted`. This format is natively supported by `HOOMD-blue <https://hoomd-blue.readthedocs.io/>`_ and is recommended for use with freud.
+Using the GSD Python API, GSD files can be easily integrated with **freud** as shown in :ref:`gettingstarted`.
+This format is natively supported by `HOOMD-blue <https://hoomd-blue.readthedocs.io/>`__ and is recommended for use with freud because it also stores other quantities, such as orientations, that are required by some analysis methods in **freud**.
+Here, we provide an example that reads data from a GSD file.
 
 .. code-block:: python
 
     import gsd.hoomd
     traj = gsd.hoomd.open('trajectory.gsd', 'rb')
 
     for frame in traj:
-        rdf.compute((frame.configuration.box, frame.particles.position), reset=False)
+        rdf.compute(frame, reset=False)
+
+MDAnalysis Readers
+------------------
+
+The `MDAnalysis <https://www.mdanalysis.org/>`__ package can read `many popular trajectory formats <https://www.mdanalysis.org/docs/documentation_pages/coordinates/init.html#supported-coordinate-formats>`__, including common output formats from CHARMM, NAMD, LAMMPS, Gromacs, Tinker, AMBER, GAMESS, and more.
+
+DCD files are among the most familiar simulation outputs due to their longevity.
+Here, we provide an example that reads data from a DCD file.
+
+.. code-block:: python
+
+    import MDAnalysis
+    reader = MDAnalysis.coordinates.DCD.DCDReader('trajectory.dcd')
+
+    for frame in reader:
+        rdf.compute(system=frame, reset=False)
+
+garnett Trajectories
+--------------------
+
+The `garnett <https://garnett.readthedocs.io/>`__ package can read `several trajectory formats <https://garnett.readthedocs.io/en/stable/readerswriters.html#file-formats>`__ that have historically been supported by the HOOMD-blue simulation engine, as well as other common types such as DCD and CIF.
+The **garnett** package will auto-detect supported file formats by the file extension.
+Here, we provide an example that reads data from a POS file.
+
+.. code-block:: python
+
+    import garnett
+
+    with garnett.read('trajectory.pos') as traj:
+        for frame in traj:
+            rdf.compute(system=frame, reset=False)
+
+OVITO Modifiers
+---------------
+
+The `OVITO Open Visualization Tool <https://www.ovito.org/>`__ supports user-written Python modifiers.
+The **freud** package can be installed alongside OVITO to enable user modifiers that leverage analyses from **freud**.
+Below is an example modifier that creates a user particle property in the OVITO pipeline for Steinhardt bond order parameters.
```

Comment:
Add a link to the [Ovito Python modifier docs](https://www.ovito.org/manual/particles.modifiers.python_script.html).